packages/relay-kit/package.json

```diff
@@ -38,8 +38,10 @@
     "@gelatonetwork/relay-sdk": "^5.5.0",
     "@safe-global/protocol-kit": "^3.0.0",
     "@safe-global/safe-core-sdk-types": "^4.0.0",
+    "@safe-global/safe-modules-deployments": "^2.0.0",
+    "alchemy-sdk": "^3.1.2",
+    "ethers": "^6.7.1",
     "permissionless": "^0.0.35",
```

Comment:
This shouldn't be necessary any more right?

Reply:
Yes, I removed permissionless and viem

We need to discuss if we want to provide the content of the providers folder as part of the sdk or as a guide in the examples. If we remove this gas calculations then i can remove the alchemy sdk as well